semantic/flatbuffers_test.go

```diff
@@ -292,6 +292,8 @@ func getFnExprFlatBuffer() (*semantic.Package, []byte) {
 	fbsemantic.FunctionExpressionAddLoc(b, exprLoc)
 	fbsemantic.FunctionExpressionAddTyp(b, funTy)
 	fbsemantic.FunctionExpressionAddTypType(b, fbsemantic.MonoTypeFun)
+	fbsemantic.FunctionExpressionAddVectorized(b, 0)
+	fbsemantic.FunctionExpressionAddVectorizedType(b, fbsemantic.ExpressionFunctionExpression)
```

Comment:
So to properly test your code, you'll need to update the massive set of nested structures below (The `want := ...` statement that describes the desired semantic graph) to include an actual function expression in the `vectorized` field.  

In addition there needs to be an actual vectorized function expression in the flatbuffer input to deserialization. It doesn't matter what the function expression is, just that it's there and you can show that it's properly deserialized.

Reply:
Thanks for helping out in writing this test case. Done!